### PR TITLE
Add open function

### PR DESCRIPTION
**Pull request by martaiborra**

### CODE REVIEW

File: tests/test_open.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+########################################################################
+#
+#       Created: May 26, 2021
```
Comment (FrancescAlted) [inline tests/test_open.py:3]: This has been created later, right? ;-). At any rate, I don't think this field has much sense, and I'd remove it from all files.

File: blosc2/SChunk.py [hunk 2]
```diff
@@ -274,7 +294,58 @@ def update_data(self, nchunk, data, copy):
...
             If some problem was detected.
         """
+        blosc2_ext._check_access_mode(self.urlpath, self.mode)
         return super(SChunk, self).update_data(nchunk, data, copy)
 
     def __dealloc__(self):
         super(SChunk, self).__dealloc__()
+
+
+def open(urlpath, mode="a",  **kwargs):
```
Comment (FrancescAlted) [inline blosc2/SChunk.py:304]: Two spaces before `**kwargs`

**Review by FrancescAlted — APPROVED** [review]
I just have some cosmetic comments.  Otherwise this looks great.

File: blosc2/SChunk.py (hunk 2 above)
Comment (martaiborra) [reply blosc2/SChunk.py]: Fixed in 4b00b0a790da9cf1578b13a6cdf761552a28fb92.

**Pull request merged**